docs/admin/kubeadm.md

```diff
@@ -203,6 +206,27 @@ There are some environment variables that modify the way that `kubeadm` works.
 | `KUBE_COMPONENT_LOGLEVEL` | `--v=4` | Logging configuration for all Kubernetes components |
 
 
+## Releases and release notes
+
+If you already have kubeadm installed and want to upgrade, run `apt-get update && apt-get upgrade` or `yum update` to get the latest version of kubeadm.
+
+ - Second release between v1.4 and v1.5: `v1.5.0-alpha.2.384+9980291736556e`
```

Comment:
Is this really the right place to document release notes?

Off the top of my head I'm not sure where else they might go, but this doc feels wrong.